lib/experimental/new_embed.dart

```diff
@@ -614,6 +615,109 @@ class FlashBox {
   void hide() {
     _element.setAttr('hidden');
   }
+
+  void show() {
+    _element.clearAttr('hidden');
+  }
+}
+
+class AnalysisResultsElement extends DElement {
```

Comment:
nit: A word like "element" doesn't do a lot to communicate what this class is doing. Consider something like "AnalysisResultsBar," "AnalyzerResultsStrip," etc.

Reply:
Renamed to AnalysisResultsController since it doesn't extend DElement anymore. Open to better names for a manager / controller class. It's managing a few different things... I'm not completely sold on the name 🤷‍♂